applications/serial_lte_modem/src/twi/slm_at_twi.c

```diff
@@ -42,10 +46,8 @@ static void do_twi_list(void)
 	memset(rsp_buf, 0, sizeof(rsp_buf));
 	sprintf(rsp_buf, "\r\n#XTWILS: ");
 
-	for (int i = 0; i < TWI_MAX_INSTANCE; i++) {
-		if (slm_twi_dev[i]) {
-			sprintf(rsp_buf + strlen(rsp_buf), "%d,", i);
-		}
+	for (size_t i = 0U; i < ARRAY_SIZE(slm_twi_dev); i++) {
+		sprintf(rsp_buf + strlen(rsp_buf), "%d,", i);
 	}
```

Comment:
This won't work with the way `slm_twi_dev` is currently initialized. I think that designated initializers should be used there.